src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -2464,6 +2467,27 @@ public function preferSafeContentData()
             ],
         ];
     }
+
+    /**
+     * @group legacy
+     */
+    public function testXForwarededAllConstantDeprecated()
+    {
+        $this->expectDeprecation('Since symfony/http-fundation 5.2: The "HEADER_X_FORWARDED_ALL" constant is deprecated, use either "HEADER_X_FORWARDED_FOR | HEADER_X_FORWARDED_HOST | HEADER_X_FORWARDED_PORT | HEADER_X_FORWARDED_PROTO" or "HEADER_X_FORWARDED_AWS_ELB" or "HEADER_X_FORWARDED_TRAEFIK" constants instead.');
+
+        Request::setTrustedProxies([], Request::HEADER_X_FORWARDED_ALL);
+    }
+
+    /**
+     * The flag 0b10000000 is used to identify the constant HEADER_X_FORWARDED_ALL
+     * To be removed in 6.0.
```

Comment:
not really as we don't want to use that position in v6 either, isn't ?

Reply:
removed.. Indeed, we don't know if we can remove it in v6